docs/release-notes/release-notes-1.0.0.md

```diff
@@ -0,0 +1,104 @@
+Cryptopower is a self-custodial multi-coin wallet application designed for both desktop and mobile platforms. It offers support for creating wallets not only for Decred but also for other cryptocurrencies such as Bitcoin and Litecoin. The app is built using [Gio](https://gioui.org/), a Golang library for the implementation of cross-platform user interfaces.
+
+## Available Features
+
+- Multi-Coin support - Native SPV wallets for Decred, Bitcoin and Litecoin.
+ - Coin Control - It allows users to select exact addresses and outputs to be used in a transaction.
+- CoinShuffle++ - CoinShuffle++ (CSPP) is a mixing protocol used to create Decred CoinJoin transactions.
+- Staking -  This allows users to purchase tickets via DCR’s [PoS (Proof-Of-Stake) consensus](https://docs.decred.org/proof-of-stake/overview/) implementation.
+- Governance - This allows users to vote on Decred Proposals, Consensus changes and Treasury Spendings.
+- Set gap limit - Users can choose a custom gap limit for use with address discovery.
+- Instant Exchange - Instant exchanging between assets using (Flyp.me, Godex, Changenow).
+- Fee Rates API - This allows for custom fee selection when sending BTC transactions.
+- Transaction note - This allows for a note/description when sending transactions.
+
+## Verifying the Release
+
+In order to verify the release, you'll need to have `gpg` or `gpg2` installed on your system. Once you've obtained a copy (and hopefully verified that as well), you'll first need to import the keys that have signed this release if you haven't done so already:
```

Comment:
This sentence is confusing especially the last part. It looks disconnected.

Reply:
how so?

Reply:
What do you think?

```
To verify the release, you must have either gpg or gpg2 installed on your system and import the keys that have signed this release. After installing gpg or gpg2, you can proceed with importing the key below if you haven't already done so:
```

Reply:
hmmm, not a bad rephrase, failing to see the issue with the original paragraph

Reply:
> you'll first need to import the keys that have signed this release if you haven't done so already

I think @Sirmorrison was referring to this line. What keys? The key is just below the sentence but we can improve there.

Reply:
please take another look

Reply:
All good.